lib/deepgram/config.js

```diff
@@ -0,0 +1,24 @@
+const transcriptionOptions = {
```

Comment:
these options should be provided by user who use this lib, don't hardcode these opts.